Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#Idea Opus Proventus\r\n\r\nprint('Hello World')\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision 2d74ea9152fa9189b8e856b744fdac5364854e21)
+++ b/main.py	(date 1708183636183)
@@ -1,3 +1,3 @@
 #Idea Opus Proventus
+from utils import *
 
-print('Hello World')
